### DISCUSSION TITLE
Issue trying to run the 2022 version

### DISCUSSION THREAD

**Discussion by inunezfe** [NREL/ReEDS-2.0#139] [Q&A: Execution] [answered]
Hello ReEDS team.

I recently pulled the 2022 version from the repository, and when i tried to ran it, i have encountered the following error when trying to run the cases_small.csv without making any changes:

<img width="1259" alt="image" src="https://user-images.githubusercontent.com/111766958/215179817-b1b6e902-612c-4340-8f83-a82cce284dc8.png">

I have traced the error up following calc financial inputs and support functions, and i think it is related to the file ivt.csv and how the vintage definition is handled.

Can you help me out to sort this issue?
Thanks

**Comment by wesleyjcole** [comment, marked as answer]
It looks like there were some new CCS technologies that were added into the model, but were not added into ivt_small.csv.  I added the technologies to the file and can now run cases_small.csv on my machine.  The change has been pushed to main, so if you pull the head of main, this should now be working.
